### PR TITLE
New Live Lab created - Leveraging Oracle Analytics Cloud with Essbase

### PR DESCRIPTION
**Pull request by srinivas-srinivasarangachari**
# Add here your WMS ID
11540

# General requirements
1. Information in the workshop is adequate and updated - Yes
2. Code is correct and working - Yes
3. Links are correct - Yes
4. Make sure you enter the help email link in your manifest.json - Yes
5. Please make sure WMS URLs are updated as needed after your PR is approved - Yes

# Checklist - Refer to the QA document for the complete list 
Please confirm that the following is completed before submitting your PR

- [Yes] All filenames are lower case (including folders, images, files, etc.)
- [Yes] Filenames are descriptive
- [ Yes] Your workshop folder structure should be similar to the one used in the sample workshop (https://github.com/oracle-livelabs/common/tree/main/sample-livelabs-templates/sample-workshop)
- [ Yes] Are you using multiple versions (desktop/, sandbox/, tenancy/)? Make sure that each of them contains a manifest.json and an index.html
- [ Yes] Image references in markdown contain an alternate text

### CODE REVIEW

File: essbase-oac/intro/intro.md
Comment (denisemyrick) [inline essbase-oac/intro/intro.md:1]: Follow markdown formatting requirements from QA. Getting Started should be Get Started.

**Review by denisemyrick — CHANGES_REQUESTED** [review]

File: essbase-oac/Connect-OAC-to-Essbase-cube/connect.md
[the remark this answers is not in the document]
Comment (srinivas-srinivasarangachari) [reply essbase-oac/Connect-OAC-to-Essbase-cube/connect.md:1]: Have made the necessary changes

**Comment by srinivas-srinivasarangachari** [comment]
Have made the necessary changes

**Review by denisemyrick — DISMISSED** [review]

**Review by ramumg — APPROVED** [review]

**Comment by ramumg** [comment]
Hi @srinivas-srinivasarangachari, 

Your branch is out of date. Please fix this so we can go ahead and merge this pull request.

**Comment by srinivas-srinivasarangachari** [comment]
Hi

 I have fixed it, could you please check.

Thanks and Regards,
Srini
________________________________
From: ramumg ***@***.***>
Sent: 23 February 2024 10:26 PM
To: oracle-livelabs/analytics-ai ***@***.***>
Cc: Srinivas Srinivasarangachari Ramaprasad ***@***.***>; Mention ***@***.***>
Subject: [External] : Re: [oracle-livelabs/analytics-ai] New Live Lab created - Leveraging Oracle Analytics Cloud with Essbase (PR #276)


Hi @srinivas-srinivasarangachari<https://urldefense.com/v3/__https://github.com/srinivas-srinivasarangachari__;!!ACWV5N9M2RV99hQ!OGTiyfVA2CcTi0giaKQ-uOqnncuZBN9BVN3XOVAQJDOtLYJPnuS2PlCyiRZmoCvSLlYMA4GfvFvN9JCoUkL67zU9wU1eW6OvGdgacg$>,

Your branch is out of date. Please fix this so we can go ahead and merge this pull request.

**Pull request merged**